README.md

```diff
@@ -78,10 +78,10 @@ NPROCS=1               # Number of processes to use during compilation PER PLATF
 docker buildx build --platform $PLATFORMs --load --build-arg NODE_VERSION=$NODE_VERSION --build-arg NPROCS=$NPROCS -t static-node:$NODE_VERSION .
 ```
 
-Then, we extract the binary for a  for a specific platform:
+Then, we extract the binary for a specific platform:
 ```
 PLATFORM='linux/arm/v7'
-APP=$(docker run --platform $PLATFORM --rm -it -d static-node:$NODE_VERSION) 
-docker cp $APP:/usr/src/app/node/out/Release/node ./node-static-armv7-$PLATFORM-$NODE_VERSION
-docker kill $APP
+IN='/usr/src/app/node/out/Release/node'
+OUT=$(echo "node-static-$PLATFORM-$NODE_VERSION" | tr / -)
+docker run --platform $PLATFORM --rm static-node:$NODE_VERSION cat "$IN" > "$OUT"
```

Comment:
While at it, I also thought that the single `docker run` command was simpler than the the three docker commands (run / cp / kill). Note that, when using `cat` in this way, the `-it` flags must not be used, otherwise it corrupts the binary file during the copy (I checked with `md5sum`).

Reply:
Maybe we should add a small caveat about the `-it` flags.

Reply:
Done. Is the note too prominent? :-)

Reply:
Looks good!

Reply:
Ah - I hadn't noticed that there were 3 occurrences of `docker cp` in the README, and this PR amended the last occurrence only. Not a problem, only a consistency matter I guess. You may eventually wish to revisit this. :-)